### DISCUSSION TITLE
showmewebcam

### DISCUSSION THREAD

**Discussion by ralphk11** [geerlingguy/pi-webcam#37] [General]
This is a great project however there is a new one that is quite good, especially if you are using a pi zero: https://github.com/showmewebcam/showmewebcam

**Comment by geerlingguy** [comment]
Agreed! I've been recommending it a bit, and at some point soon, I'm going to be doing a follow up video reviewing it and comparing it to some other options.

**Comment by mummson** [comment]
Is there any reason to use showmewebcam over pi-webcam? or vice versa?

**Reply by geerlingguy** [reply]
Showmewebcam runs faster, is a little simpler, and likely uses less energy to have a webcam running continuously. I'd probably recommend it unless you want to also have the Pi running like a regular Pi with the ability to log in and run other things on it or SSH into it.

**Reply by mummson** [reply]
Ahh ok, ill have to check that out then :)
